Automattic/jetpack
# Build: Replace Old Packages

#### Changes proposed in this Pull Request:

These packages have not been updated in years, and include old versions of dependencies that we should get rid of:
* `gulp-delete-lines` → `gulp-rm-lines`
* `gulp-modify` → `gulp-modify-css-urls`
* `gulp-banner` → `gulp-append-prepend`

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
N/A

#### Testing instructions:
Before applying patch:
1. `./node_modules/.bin/gulp languages:build`
2. `mkdir old-langs`
3. `cp languages/json/* old-langs/`
4. `./node_modules/.bin/gulp frontendcss`
5. `mkdir old-css`
6. `cp css/* old-css/`

After applying patch:
1. `./node_modules/.bin/gulp languages:build`
2. `mkdir new-langs`
3. `cp languages/json/* new-langs/`
4. `./node_modules/.bin/gulp frontendcss`
5. `mkdir new-css`
6. `cp css/* new-css/`
7. `diff -r old-langs new-langs`. Verify there are no differences.
7. `diff -r old-css new-css`. Verify there are no differences.

#### Proposed changelog entry for your changes:
None needed.
